CODEOWNERS

```diff
@@ -2,3 +2,4 @@
 /proposed/security* @php-fig/psr-9-10
 /proposed/phpdoc* @php-fig/psr-5
 /proposed/clock* @php-fig/psr-20
+/proposed/internationalization* @php-fig/psr-21
```

Comment:
This group has yet to be created here: https://github.com/orgs/php-fig/teams

Reply:
Group is already created and members added